package.json

```diff
@@ -4,11 +4,16 @@
   "packageManager": "pnpm@9.12.3+sha512.cce0f9de9c5a7c95bef944169cc5dfe8741abfb145078c0d508b868056848a87c81e626246cb60967cbd7fd29a6c062ef73ff840d96b3c86c40ac92cf4a813ee",
   "scripts": {
     "build": "turbo build",
-    "test": "turbo test",
+    "test": "pnpm test:affected",
+    "test:affected": "turbo test --affected --output-logs=new-only",
+    "test:all": "turbo test",
+    "test:all:force": "turbo test --force",
+    "test:ci": "turbo test --concurrency=5",
```

Comment:
Does it work with 7 or 8 or 9?

Reply:
Maybe, but this didn't impact execution time at all anyway